lib/api/request/kuzzleRequest.ts

```diff
@@ -30,7 +30,8 @@ import { KuzzleError, InternalError } from '../../kerror/errors';
 import kerror from '../../kerror';
 import { Deprecation, User } from '../../types';
 import * as assert from '../../util/assertType';
-import { get, isPlainObject } from '../../util/safeObject';
+import { isPlainObject } from '../../util/safeObject';
+import * as _ from 'lodash';
```

Comment:
this import the whole library, you could just import the single method with:

`import get from 'lodash/get';`

Reply:
I agree on this for a frontend application but I'm not sure it changes something for a Node.js application.  
I think all our dependencies should use this kind of partial import maybe. Either way, actually the whole lodash code is always loaded into RAM :facepalm: 
https://user-images.githubusercontent.com/4447392/139255742-08e819d6-5dff-4439-b341-f48004ddb45b.png